main.css

```diff
@@ -0,0 +1,83 @@
+
+ 
+body{
+font-family: 'Merriweather', serif;
+margin: 0;
+
+}
+.main-nav {
+    padding: 20px;
+    font-size:20px;
+    border: solid black 1px;
+    background-color: lightseagreen;
+ 
+  
+}
+
+.nav-link {
+    padding-right: 20px;
+    color: black;
+    text-decoration: none;
+   
+}
+
+
+.nav-link:last-of-type {
+    padding-right: 0;
+    width: 30px;
```

Comment:
why does this one need a `width`? note also that if you inspect this element in the browser, the 30px width isn't being respected because the content is too big to fit in a 30px box